docs/pages/access-controls/guides/access-requests-slack.mdx

```diff
@@ -0,0 +1,347 @@
+    ```
+  </TabItem>
+  <TabItem label="From Source">
+    To install from source you need `git` and `go >= (=teleport.golang=)` installed.
+
+    ```bash
+    # Checkout teleport-plugins
+    git clone https://github.com/gravitational/teleport-plugins.git
+    cd teleport-plugins/access/slack
+    make
+    ```
+  </TabItem>
+</Tabs>
+
+<Admonition type="tip">
+  View the [Slack Plugin README](https://github.com/gravitational/teleport-plugins/tree/master/access/slack).
+</Admonition>
+
+### Configuring Teleport Slack
+
+Teleport Slack uses a config file in `TOML` format.
+
+Generate a boilerplate config by running the following command:
+
+```bash
+teleport-slack configure > teleport-slack.toml
+```
+
+#### Copy certificates
+
+Navigate back to the certificates you generated with `tctl auth sign` before. The plugin installer creates a folder for them at: `/var/lib/teleport/plugins/slack/`. 
+
+Copy or move the certificates there and point the config file to them:
+
+```bash
+# Navigate up directories
+cd ../
+# Move certificate files created by 'tctl auth sign'
+sudo mv -f auth.cas /var/lib/teleport/plugins/slack/
+sudo mv -f auth.crt /var/lib/teleport/plugins/slack/
+sudo mv -f auth.key /var/lib/teleport/plugins/slack/
+```
+
+#### Editing the config file
+
+Run the following command to modify the configuration file before we deploy it:
+
+```bash
+sudo nano teleport-access-slack/teleport-slack.toml
+```
+
+Teleport YAML configuration files are divided into sections:
+
+```conf
+(!examples/resources/plugins/teleport-slack.toml!)
+```
+
+1. Find the "Teleport" section and add the certificates you generated with `tctl auth sign` before.
+2. Find the "Slack" section and add the OAuth token, signing token, and the desired channel name.
+3. Add the desired recipients. 
+
+  For users, add the email address associated with their Slack profile. 
+  
+  For channels, add the Slack Channel ID (which can be found in the URL path; `YYYYYYY` below).
+
+  ![Find Slack Channel ID](../../../img/enterprise/plugins/slack/slack-channel-id.png)
+
+4. Copy `teleport-slack.toml` to the location used by the Slack Plugin:
+
+   ```bash
+   sudo mv teleport-access-slack/teleport-slack.toml /etc/teleport-slack.toml
+   ```
+
+## Step 4/4. Test run
+
+Assuming that Teleport is running, and you've created the Slack app, the plugin config, and provided all the certificates — you can now run the plugin and test the workflow!
+
+```bash
+sudo teleport-slack start
+```
+
+If everything works fine, the log output should look like this:
+
+```bash
+sudo teleport-slack start
+# INFO   Starting Teleport Access Slack Plugin (=teleport.plugin.version=): slack/app.go:72
+```
+
+### Create a test user
+
+Many of Teleport's default user roles lack the ability to make role requests.
+
+Create a test role called `dev` with that ability:
+
+```yaml
+kind: "role"
+version: "v3"
+metadata:
+  name: "dev"
+  namespace: "default"
+spec:
+  allow:
+    request:
+      roles: ['*']
```

Comment:
This is a bit too permissive, let's scope to one role only, for example, like here:

https://github.com/gravitational/teleport/blob/master/docs/pages/access-controls/guides/dual-authz.mdx#dual-authorization-1

But without thresholds